core/src/main/java/de/evoila/cf/broker/service/impl/DeploymentServiceImpl.java

```diff
@@ -167,6 +167,21 @@ public ServiceInstanceOperationResponse updateServiceInstance(String serviceInst
         return serviceInstanceOperationResponse;
     }
 
+    @Override
+    public ServiceInstanceOperationResponse updateServiceInstanceContext(String serviceInstanceId,
+                                                                         ServiceInstanceUpdateRequest serviceInstanceUpdateRequest)
+            throws ServiceBrokerException, ServiceInstanceDoesNotExistException, ServiceDefinitionDoesNotExistException {
+
+        ServiceInstance serviceInstance = serviceInstanceRepository.getServiceInstance(serviceInstanceId);
+        if (serviceInstance == null) {
+            throw new ServiceInstanceDoesNotExistException(serviceInstanceId);
+        }
+        serviceInstance.setContext(serviceInstanceUpdateRequest.getContext());
```

Comment:
Do you know whether the platform will send a full context object or can it maybe send just a new space_id to change only that value? In that case our context would be missing essential fields after the update.

Reply:
I have found no word on this in the spec.